behaviour/graql/language/delete.feature

```diff
@@ -732,13 +1378,14 @@ Feature: Graql Delete Query
 
 
   Scenario: delete everything in a complex pattern
```

Comment:
I'm not sure this counts as an "edge" case - this is more advanced or complete usage of delete :)

Reply:
These delete tests were here already and I don't really know what they do, or how to categorise them.

Reply:
Moved to a new section named Complex Patterns.